### PR TITLE
Move ConsoleLogger.v1 to spacy-legacy

### PR DESCRIPTION
**Pull request by thomashacker**
This `PR` moves the `ConsoleLogger.v1` to spacy-legacy. 
The upcoming `ConsoleLogger.v2` will feature saving training logs to a `JSONL` file (and more ✨).

### CODE REVIEW

**Comment by adrianeboyd** [comment]
Can you also add it to the tests here?

https://github.com/explosion/spacy-legacy/blob/master/spacy_legacy/tests/test_legacy.py

File: spacy_legacy/tests/test_legacy.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
     ("architectures", "Tok2Vec.v1"),
     ("layers", "StaticVectors.v1"),
     ("loggers", "WandbLogger.v1"),
+    ("loggers", "ConsoleLogger.v1"),
```
Comment (svlandeg) [inline spacy_legacy/tests/test_legacy.py:18]:
```suggestion
    ("loggers", "ConsoleLogger.v1"),
    ("loggers", "WandbLogger.v1"),
```
just to keep this list alphabetically

File: setup.cfg [hunk 2]
```diff
@@ -48,6 +48,7 @@ spacy_architectures =
     spacy-legacy.TransitionBasedParser.v1 = spacy_legacy.architectures.parser:TransitionBasedParser_v1
 spacy_loggers =
     spacy-legacy.WandbLogger.v1 = spacy_legacy.loggers:wandb_logger_v1
+    spacy-legacy.ConsoleLogger.v1 = spacy_legacy.loggers:console_logger_v1
```
Comment (svlandeg) [inline setup.cfg:51]:
```suggestion
    spacy-legacy.ConsoleLogger.v1 = spacy_legacy.loggers:console_logger_v1
    spacy-legacy.WandbLogger.v1 = spacy_legacy.loggers:wandb_logger_v1
```
Also just to keep alphabetically

**Review by svlandeg — COMMENTED** [review]
Otherwise looks good :-)

**Pull request merged**